src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1254,6 +1268,7 @@ private void Keyword_PropertyChanged(object sender, PropertyChangedEventArgs e)
             }
         }
 
+        [Obsolete]
```

Comment:
Please add a message saying this will be removed in a future version of Dynamo, as we do with all obsoleted APIs.